### PR TITLE
Add methods to restore nuget packages in vs code article and update c…

### PR DESCRIPTION
**Pull request by didiyordanova**
…ommands in sample apps.

### CODE REVIEW

File: demos-and-sample-apps/controls-showcase-app.md [hunk 1]
```diff
@@ -47,17 +47,15 @@ You can access the Controls Samples application in the following ways:
...
 	* ControlsSamples application on MacOS.
 	
 	![Telerik UI for .NET MAUI ControlsSamples App MacCatalyst](images/controlssamples-macos.png)
 
 1. Run on iOS using the following command:
 
- `dotnet build -t:Run -f net7.0-ios -p:_DeviceName=:v2:udid=02C556DA-64B8-440B-8F06-F8C56BB7CC22`
- 
- where the `-p:_DeviceName=:v2:uuid==02C556DA-64B8-440B-8F06-F8C56BB7CC22` is the desired device id.
+ `dotnet build -t:Run -f net8.0-ios` - this command starts a default simulator. If you want to run the application on specific device then you need to specify the device id by running the following command: `dotnet build -t:Run -f net8.0-ios -p:_DeviceName=:v2:udid=02C556DA-64B8-440B-8F06-F8C56BB7CC22` where the device id is `02C556DA-64B8-440B-8F06-F8C56BB7CC22`.
```
Comment (ikoevska) [inline demos-and-sample-apps/controls-showcase-app.md:58]:
```suggestion
 `dotnet build -t:Run -f net8.0-ios`&mdash;Тhis command starts a default simulator. To run the application on specific device, you need to provide the device ID. For example: `dotnet build -t:Run -f net8.0-ios -p:_DeviceName=:v2:udid=02C556DA-64B8-440B-8F06-F8C56BB7CC22` where the device ID is `02C556DA-64B8-440B-8F06-F8C56BB7CC22`.
```

Comment (ikoevska) [reply demos-and-sample-apps/controls-showcase-app.md:58]: In addition, how can users get the device ID for the simulator?

File: demos-and-sample-apps/crypto-app.md [hunk 2]
```diff
@@ -52,17 +52,16 @@ You can access the SDKBrowser application in the following ways:
...
 	* CryptoTracker application on MacOS.
 	
 	![Telerik UI for .NET MAUI CryptoTracker App MacCatalyst](../images/cryptotracker-macos.png)
 
 1. Run on iOS using the following command:
 
- `dotnet build -t:Run -f net7.0-ios -p:_DeviceName=:v2:udid=02C556DA-64B8-440B-8F06-F8C56BB7CC22`
- 
- where the `-p:_DeviceName=:v2:uuid==02C556DA-64B8-440B-8F06-F8C56BB7CC22` is the desired device id.
+ `dotnet build -t:Run -f net8.0-ios` - this command starts a default simulator. If you want to run the application on specific device then you need to specify the device id by running the following command: `dotnet build -t:Run -f net8.0-ios -p:_DeviceName=:v2:udid=02C556DA-64B8-440B-8F06-F8C56BB7CC22` where the device id is `02C556DA-64B8-440B-8F06-F8C56BB7CC22`.
```
Comment (ikoevska) [inline demos-and-sample-apps/crypto-app.md:63]: Same notes as `demos-and-sample-apps/controls-showcase-app.md`

File: demos-and-sample-apps/sdkbrowser-app.md [hunk 3]
```diff
@@ -41,11 +41,11 @@ You can access the SDKBrowser application in the following ways:
...
 1. Run on macOS using the following command:
 
- `dotnet build -t:Run -f net7.0-maccatalyst`
+ `dotnet build -t:Run -f net8.0-maccatalyst`
  
-1. Run on iOS using the following command: Append the value to the parameter `-p:_DeviceName=:v2:uuid=:`
+1. Run on iOS using the following command:
 
- `dotnet build -t:Run -f net7.0-ios -p:_DeviceName=:v2:udid=02C556DA-64B8-440B-8F06-F8C56BB7CC22`
+ `dotnet build -t:Run -f net8.0-ios` - this command starts a default simulator. If you want to run the application on specific device then you need to specify the device id by running the following command: `dotnet build -t:Run -f net8.0-ios -p:_DeviceName=:v2:udid=02C556DA-64B8-440B-8F06-F8C56BB7CC22` where the device id is `02C556DA-64B8-440B-8F06-F8C56BB7CC22`.
```
Comment (ikoevska) [inline demos-and-sample-apps/sdkbrowser-app.md:48]: Same notes as `demos-and-sample-apps/controls-showcase-app.md`

File: installation/vs-code-integration/new-project-wizard.md [hunk 4]
```diff
@@ -30,6 +30,43 @@ Next in the menu, look for and select **Telerik UI for .NET MAUI Template Wizard
 
 ![Telerik UI for .NET MAUI VS Code snippets](./images/MauiTemplateWizard.png)
 
+## Set up the Telerik NuGet Server
+
+You can use one of the following options to configure the needed credentials for your telerik license (trial or commersial):
```
Comment (ikoevska) [inline installation/vs-code-integration/new-project-wizard.md:35]:
```suggestion
You can use one of the following options to configure the required credentials for your Telerik license (trial or commercial):
```

File: installation/vs-code-integration/new-project-wizard.md [hunk 5]
```diff
@@ -30,6 +30,43 @@ Next in the menu, look for and select **Telerik UI for .NET MAUI Template Wizard
...
+
+### Using the .NET CLI
+
+When adding NuGet sources from the .NET CLI, the credentials are stored inside the `Nuget.Config` file. To add the Telerik NuGet package source with the .NET CLI, use the `dotnet nuget add source` command. This command creates or updates the `NuGet.Config` file for you, so you don't have to edit it manually. For more details on this case, review the [Restoring NuGet Packages in your CL Workflow]({%slug nuget-keys%}) article.
```
Comment (ikoevska) [inline installation/vs-code-integration/new-project-wizard.md:39]:
```suggestion

Run `dotnet nuget add source`.

When you are adding NuGet sources from the .NET CLI, the credentials are stored inside the `Nuget.Config` file. This command creates or updates the `NuGet.Config` file with your Telerik license credentials for you, so you don't have to edit it manually.

See also: [Restoring NuGet Packages in Your CI Workflow]({%slug nuget-keys%}).
```

File: installation/vs-code-integration/new-project-wizard.md [hunk 6]
```diff
@@ -30,6 +30,43 @@ Next in the menu, look for and select **Telerik UI for .NET MAUI Template Wizard
...
+
+### Editing the NuGet.Config file
```
Comment (ikoevska) [inline installation/vs-code-integration/new-project-wizard.md:41]:
```suggestion
### Editing the NuGet.Config File
```

File: installation/vs-code-integration/new-project-wizard.md [hunk 7]
```diff
@@ -30,6 +30,43 @@ Next in the menu, look for and select **Telerik UI for .NET MAUI Template Wizard
...
+
+Nuget package sources and other settings are stored in a `NuGet.Config` file. You can read more about the file structure in the Microsoft article: [Nuget.Config References](https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file). To edit the file and add the Telerik feed, you need to:
```
Comment (ikoevska) [inline installation/vs-code-integration/new-project-wizard.md:43]:
```suggestion
NuGet package sources and other settings are stored in a `NuGet.Config` file. You can read more about the file structure in the Microsoft article: [NuGet.Config References](https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file). To edit the file and add the Telerik feed, you need to: 
```

File: installation/vs-code-integration/new-project-wizard.md [hunk 8]
```diff
@@ -30,6 +30,43 @@ Next in the menu, look for and select **Telerik UI for .NET MAUI Template Wizard
...
+
+### Using the .NET CLI
+
+When adding NuGet sources from the .NET CLI, the credentials are stored inside the `Nuget.Config` file. To add the Telerik NuGet package source with the .NET CLI, use the `dotnet nuget add source` command. This command creates or updates the `NuGet.Config` file for you, so you don't have to edit it manually. For more details on this case, review the [Restoring NuGet Packages in your CL Workflow]({%slug nuget-keys%}) article.
+
+### Editing the NuGet.Config file
+
+Nuget package sources and other settings are stored in a `NuGet.Config` file. You can read more about the file structure in the Microsoft article: [Nuget.Config References](https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file). To edit the file and add the Telerik feed, you need to: 
+
+1. Ensure you editing the correct and desired `.config` file. You can also create a new one with the `dotnet new nugetconfig` command.
```
Comment (ikoevska) [inline installation/vs-code-integration/new-project-wizard.md:45]:
```suggestion
1. Ensure you are editing the correct and desired `.config` file. You can also create a new one with the `dotnet new nugetconfig` command. 
```

File: installation/vs-code-integration/new-project-wizard.md [hunk 9]
```diff
@@ -30,6 +30,43 @@ Next in the menu, look for and select **Telerik UI for .NET MAUI Template Wizard
...
+When adding NuGet sources from the .NET CLI, the credentials are stored inside the `Nuget.Config` file. To add the Telerik NuGet package source with the .NET CLI, use the `dotnet nuget add source` command. This command creates or updates the `NuGet.Config` file for you, so you don't have to edit it manually. For more details on this case, review the [Restoring NuGet Packages in your CL Workflow]({%slug nuget-keys%}) article.
+
+### Editing the NuGet.Config file
+
+Nuget package sources and other settings are stored in a `NuGet.Config` file. You can read more about the file structure in the Microsoft article: [Nuget.Config References](https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file). To edit the file and add the Telerik feed, you need to: 
+
+1. Ensure you editing the correct and desired `.config` file. You can also create a new one with the `dotnet new nugetconfig` command. 
+1. Add the Telerik package source to the `.config` file. 
+
+Example of sample `NuGet.Config` file:
```
Comment (ikoevska) [inline installation/vs-code-integration/new-project-wizard.md:48]:
```suggestion
Here is a sample `NuGet.Config` file:
```

File: installation/vs-code-integration/new-project-wizard.md [hunk 10]
```diff
@@ -30,6 +30,43 @@ Next in the menu, look for and select **Telerik UI for .NET MAUI Template Wizard
...
+<configuration>
+    <packageSources>
+    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
+    <clear />
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
+    <add key="TelerikOnlineFeed" value="https://nuget.telerik.com/v3/index.json" />
+    </packageSources>
+    <packageSourceCredentials>
+    <TelerikOnlineFeed>
+        <add key="Username" value="<TELERIK EMAIL or api-key>" />
```
Comment (ikoevska) [inline installation/vs-code-integration/new-project-wizard.md:61]: Why is `api-key` lowercase here and uppercase below?

Comment (didiyordanova) [reply installation/vs-code-integration/new-project-wizard.md:61]: we use this structure, example here: https://docs.telerik.com/devtools/maui/installation/using-nuget-keys#using-a-nugetconfig-file-with-your-projects following this: https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file

**Pull request merged**